autoload/lsp.vim

```diff
@@ -475,7 +475,7 @@ function! lsp#default_get_supported_capabilities(server_info) abort
     \       'completion': {
     \           'dynamicRegistration': v:false,
     \           'completionItem': {
-    \              'documentationFormat': ['plaintext'],
+    \             'documentationFormat': ['markdown'],
```

Comment:
since array is ordered can we instead use `['markdown', 'plaintext']`

Reply:
Thank you. Fixed.